### DISCUSSION TITLE
Visual indicator in TOC

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [nunocoracao/blowfish#375] [Ideas]
I think it would be awesome if the table of contents (when seen on the right hand side of an article) would get some sort of visual indication of where in the article you are currently reading. For example, if you've scrolled past header 2, 3 or 4 the corresponding TOC headers can be highlighted.

Unfortunately, I'm no CSS/frontend wizard, so I'm not sure where to even begin with this. Figured I'd just throw this out there. 😄

**Comment by nunocoracao** [comment]
Better to open a feature request ticket see if it gets any votes. I think is a good idea... not super important at this stage.
